### DISCUSSION TITLE
Install and run swc without npm and node

### DISCUSSION THREAD

**Discussion by Canadadry** [swc-project/swc#4312] [Q&A] [answered]
I have a small js project were we don't use npm or node. You old jquery website Don't judge please). And I would like to setup a small bundle and minification phase without have to install a lot of tool and write a lot of config file. So I pick this project which seems to match my needs but I can't find any usefull info on how to install it and run it wihtout npm.

I tried the pre build binaries but I cannot launch it it gave me : 

```bash
> ./swc.darwin-arm64.node 
Failed to execute process './swc.darwin-arm64.node'. Reason:
exec: Exec format error
The file './swc.darwin-arm64.node' is marked as an executable but could not be run by the operating system.
```
I was expecting that because of the `.node` at the end. 

Does someone has an idea ?

**Comment by kdy1** [comment, marked as answer]
Some parts of swc cli is published as https://crates.io/crates/swc_cli

**Reply by Canadadry** [reply]
thanks,
